### PR TITLE
[IntraHealth] New dashboards

### PR DESCRIPTION
**Pull request by panprosiak**
Now 'Valeur des Stocks PNA Disponible' and 'Recap Passage 2' are also operating on test data.

### CODE REVIEW

**Comment by MaciejChoromanski** [comment]
Now 'Taux de Perte' and 'Taux de Peremption' are operating on the test data

**Comment by MaciejChoromanski** [comment]
Now all of the dashboards operate on test data, also we've made some bug fixes and updates

File: custom/intrahealth/sqldata.py [hunk 1]
```diff
@@ -5395,3 +5502,482 @@ def get_headers(self):
...
+    def desired_location(self):
+        if self.config['selected_location']:
+            return self.config['selected_location']
+        return None
+
+    @property
+    def month(self):
+        if self.config['month']:
+            return int(self.config['month'])
+        return 01
```
[the remark this answers is not in the document]
Comment (hamelDimagi) [reply custom/intrahealth/sqldata.py]: @MaciejChoromanski This line is also causing tests to fail (see log in: https://travis-ci.org/dimagi/commcare-hq/jobs/575307146)

[the remark this answers is not in the document]
Comment (MaciejChoromanski) [reply custom/intrahealth/sqldata.py]: @hamelDimagi I've changed it

**Review by gbova — APPROVED** [review]
Looks good to me! None of my comments are blocking.

**Pull request merged**